packages/core/test/CardanoNode/mocks.ts

```diff
@@ -69,7 +69,7 @@ export const healthCheckResponseMock = (opts?: {
       hash: opts?.hash ?? '9ef43ab6e234fcf90d103413096c7da752da2f45b15e1259f43d476afd12932c',
       slot: opts?.slot ?? 52_819_355
     },
-    networkSync: opts?.networkSync ?? Cardano.Percent(0.999)
+    networkSync: opts?.networkSync ?? Cardano.Percent(99.9)
```

Comment:
`Cardano.Percent` type should only be used with original format. Please create a new opaque number type, or just use `number` for `networkSync`.
```suggestion
    networkSync: opts?.networkSync ?? 99.9
```